edge-hub/test/Microsoft.Azure.Devices.Routing.Core.Test/Microsoft.Azure.Devices.Routing.Core.Test.csproj

```diff
@@ -26,6 +26,11 @@
     <OutputPath>bin\CodeCoverage</OutputPath>
     <DefineConstants>DEBUG;TRACE</DefineConstants>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="util\concurrency\**" />
```

Comment:
Why do you need this?

Reply:
I don't. Seems like I 'exclude'-ed the empty concurrency folder from the project instead of 'delete'-ing it in visual studio. Fixed by the next commit.